### PR TITLE
Contribution 29/07 - v2.1.0

### PR DESCRIPTION
**Pull request by Raphy**

### CODE REVIEW

File: tests/Majora/Installer/Tests/IntegrationTest.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Majora\Installer\Tests;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+class IntegrationTest extends \PHPUnit_Framework_TestCase
```
Comment (Nyxis) [inline tests/Majora/Installer/Tests/IntegrationTest.php]: Php docs

**Comment by Nyxis** [comment]
SQUASH !!! ♻️

**Comment by Raphy** [comment]
Commits are squashed

**Comment by Nyxis** [comment]
👍

**Pull request merged**